### ISSUE TITLE
Problem with Source Exeter City Council, UK

### ISSUE THREAD

**Issue by AtomBrake** [mampfes/hacs_waste_collection_schedule#1145]
There seems to be an error with Exeter City Council where the API reply is not in the expected format, log entries as follows:
```
fetch failed for source Exeter City Council: Traceback (most recent call last): File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source_shell.py", line 134, in fetch entries = self._source.fetch() ^^^^^^^^^^^^^^^^^^^^ File "/config/custom_components/waste_collection_schedule/waste_collection_schedule/source/exeter_gov_uk.py", line 44, in fetch date=datetime.strptime(re.compile(REGEX_ORDINALS).sub("",d.text),"%A, %d%B %Y").date(), ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/usr/local/lib/python3.11/_strptime.py", line 568, in _strptime_datetime tt, fraction, gmtoff_fraction = _strptime(data_string, format) ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ File "/usr/local/lib/python3.11/_strptime.py", line 349, in _strptime raise ValueError("time data %r does not match format %r" % ValueError: time data 'Monday, 21Augu2023' does not match format '%A, %d%B %Y'
```

**Comment by dt215git** [comment]
Fixed in PR #1151.

**Comment by 5ila5** [comment]
fixed by #1151

will be part of release 1.42.0 or is available now as master/git version (up to date clone of this repository)
